packages/cfpb-typography/src/atoms/tagline.less

```diff
@@ -0,0 +1,52 @@
+.a-tagline {
+    display: grid;
+    grid-template-columns: 22px 1fr;
+    grid-column-gap: 10px;
+
+    // This'll target the USA flag.
+    & > *:first-child {
+        margin-top: 1px;
+    }
+
+    &_usa {
```

Comment:
Should this be a reusable utility class similar to `.u-clearfix`(e.g. `.u-nowrap` )?

Reply:
MMmmmmmm I like it. Fixed in https://github.com/cfpb/design-system/pull/1301/commits/4f615970bd6ccf87bd1bf3f5983b135b60cdc131

Reply:
Now the tagline is just a container for arbitrary text content, which may contain markup and a `no-wrap` class. I like it better.